package.json

```
@@ -1,5 +1,6 @@
 {
   "name": "outline-client",
+  "version": "1.7.3",
```

Why are you adding the version here? It's dead code, as far as I can tell.

---

It is required by `npx generate-license-file`, if we don't have version it will print the following error:

```
✖ Cannot find the "version" key in the package.json!
```

---

Yes, let's use 0.0.0
```suggestion
  "version": "0.0.0",
```

---

Done, also added a `version` comment in package.json as well.